forc/src/utils/defaults.rs

```diff
@@ -11,7 +11,6 @@ license = "Apache-2.0"
 name = "{project_name}"
 
 [dependencies]
```

Comment:
I was debating whether to keep or remove `[dependencies]`. I'm open to opinions. Keeping it makes it easier to add user libraries (ABIs for example) as dependencies. Removing it is cleaner.

Reply:
I prefer keeping it. Less typing for the user.